README.md

```diff
@@ -4,8 +4,8 @@ This repository is for running uncalibrated GLM models of lake temperatures.
 -----------------
 ## Dependent files from [`lake-temperature-model-prep pipeline`](https://github.com/USGS-R/lake-temperature-model-prep)
 _Files that will eventually be transferred using GLOBUS:_
-  * Lake - GCM cell crosswalk: `'1_prep/in/lake_cell_xwalk.csv'`
-    * Created within [`gcm_driver_data_munge_pipeline` branch](https://github.com/USGS-R/lake-temperature-model-prep/tree/gcm_driver_data_munge_pipeline) of repo
+  * Lake - GCM cell tile crosswalk: `'1_prep/in/lake_cell_tile_xwalk.csv'`
+    * Created within [the `targets` sub-pipeline](https://github.com/USGS-R/lake-temperature-model-prep/blob/main/_targets.R#L194-L213) of the repo
```

Comment:
Linking to a specific line in the repo in your documentation is a bit risky because the code could change and break your reference, especially if we add/adjust targets above that one linked. Perhaps this would be more robust to future changes?

```suggestion
    * Created within [the `targets` sub-pipeline](https://github.com/USGS-R/lake-temperature-model-prep/blob/main/_targets.R), look for the `lake_cell_tile_xwalk_df` target
```